### PR TITLE
[BA-5652] Updated all-purpose-mess-remover.dot and all-purpose-mess-remover.dot.png

### PR DESCRIPTION
**Pull request by ichengchang**
Updated all-purpose-mess-remover.dot and all-purpose-mess-remover.dot.png with submission_print_out.sh.

### CODE REVIEW

**Comment by danbills** [comment]
Could you paste generated image here?

**Comment by aednichols** [comment]
Chris showed me this the other day, if you go to the diff and click "rich diff" in the top right you get an image preview

rich diff

**Comment by cjllanwarne** [comment]
You can also go direct to the page containing the rendered image: https://github.com/broadinstitute/cromwell/tree/BA-5652_suspicious_submissions_playbook/processes/troubleshooting

File: processes/troubleshooting/all-purpose-mess-remover.dot [hunk 1]
```diff
@@ -7,10 +7,11 @@ digraph {
...
   restart_cromwell_instance [shape=oval label="Restart Cromwell's 'writer' instance"];
 
-  determine_time [shape=oval label="Determine what time things started going wrong"];
-  determine_submissions_of_interest [shape=oval label="Determine a submission of interest from around that time"];
+  download_submission_print_out [shape=oval label="Download submission_print_out.sh from github repo dsp-scripts/cromwell"];
+  make_sure_gcloud_use_firecloud [shape=oval label="Make sure gcloud is using a firecloud.org account configuration"];
+  determine_submissions_of_interest [shape=oval label="Determine a submission of interest at a time right before alerts are generated.\n\nRun the script as\n\n./submission_print_out.sh <env>\n\nSubstitute <env> with environment { dev, prod }.\n\nThe script outputs submission details in the format\n\n<submission timestamp> <submitter email> <namespace> <submission id>"];
 
-  place_submissions_on_hold [shape=oval label="Place all workflows from that submission on hold in the database"];
+  place_submissions_on_hold [shape=oval label="Place all workflows from that submission on hold in the database.\nRestart cromwell."];
```
Comment (mcovarr) [inline processes/troubleshooting/all-purpose-mess-remover.dot]: I think this line should be reverted, the "Restart Cromwell" instruction in `place_submissions_on_hold` appears to be redundant to the instruction in `restart_cromwell_instance `

File: processes/troubleshooting/all-purpose-mess-remover.dot [hunk 2]
```diff
@@ -7,10 +7,11 @@ digraph {
   # Always start with a restart:
   restart_cromwell_instance [shape=oval label="Restart Cromwell's 'writer' instance"];
 
-  determine_time [shape=oval label="Determine what time things started going wrong"];
-  determine_submissions_of_interest [shape=oval label="Determine a submission of interest from around that time"];
+  download_submission_print_out [shape=oval label="Download submission_print_out.sh from github repo dsp-scripts/cromwell"];
+  make_sure_gcloud_use_firecloud [shape=oval label="Make sure gcloud is using a firecloud.org account configuration"];
+  determine_submissions_of_interest [shape=oval label="Determine a submission of interest at a time right before alerts are generated.\n\nRun the script as\n\n./submission_print_out.sh <env>\n\nSubstitute <env> with environment { dev, prod }.\n\nThe script outputs submission details in the format\n\n<submission timestamp> <submitter email> <namespace> <submission id>"];
```
Comment (mcovarr) [inline processes/troubleshooting/all-purpose-mess-remover.dot]: TOL Not sure if this instruction should be so much more detailed than the others?

Comment (danbills) [reply processes/troubleshooting/all-purpose-mess-remover.dot]: Think we should remove output format section and add it as header to script output

**Comment by cjllanwarne** [comment]
At the top of the page containing this image is:
> Note: These processes contain shorthand descriptions for various tasks. If you aren't sure how to achieve any of these steps, look for the details in the Cromwell playbook.

I personally think that's a good balance. The diagrams show the flow through the process, and the long text contains the details of how to achieve each step. What do you think?

File: processes/troubleshooting/all-purpose-mess-remover.dot (hunk 1 above)
Comment (ichengchang) [reply processes/troubleshooting/all-purpose-mess-remover.dot]: Latest commit removed the redundancy.

**Comment by ichengchang** [comment]
all-purpose-mess-remover dot

**Comment by aednichols** [comment]
Removing hold label because it was applied for Ivan's vacation

File: processes/troubleshooting/all-purpose-mess-remover.dot [hunk 3]
```diff
@@ -7,10 +7,11 @@ digraph {
   # Always start with a restart:
   restart_cromwell_instance [shape=oval label="Restart Cromwell's 'writer' instance"];
 
-  determine_time [shape=oval label="Determine what time things started going wrong"];
-  determine_submissions_of_interest [shape=oval label="Determine a submission of interest from around that time"];
+  download_submission_print_out [shape=oval label="Download submission_print_out.sh from github repo dsp-scripts/cromwell"];
+  make_sure_gcloud_use_firecloud [shape=oval label="Make sure gcloud is using a firecloud.org account configuration"];
+  determine_submissions_of_interest [shape=oval label="Determine a submission of interest at a time right before alerts are generated"];
```
Comment (danbills) [inline processes/troubleshooting/all-purpose-mess-remover.dot]: Please add `Run the script as\n\n./submission_print_out.sh  <env>` as you had it previously

**Pull request merged**